src/repository/vocabulary.py

```diff
@@ -6,37 +6,64 @@
 from azure.cosmos import exceptions, CosmosClient, PartitionKey
 from azure.identity import DefaultAzureCredential
 import uuid
-from src.services.languageDetection import detect_language_code
+from services.detect_language import detect_language_code
 
 """
 Creating connection to CosmosDB
 """
 load_dotenv()
 credential = DefaultAzureCredential()
 cosmos_endpoint = os.getenv("COSMOS_ACCOUNT_URI")
+if not cosmos_endpoint:
+    raise ValueError("COSMOS_ACCOUNT_URI is not set")
 cosmos_client = CosmosClient(cosmos_endpoint, credential=credential)
 dictionary_database = cosmos_client.get_database_client("dictionary")
 words_container = dictionary_database.get_container_client("words")
+user_container = dictionary_database.get_container_client("users")
 
 
-def get_all_words(user_id):
+def get_or_create_user_id_in_DB(telegram_id: str, username: str, first_name: str, last_name: str | None) -> str:
```

Comment:
I don't like the `in_DB` ending here. You don't have it in the other methods. I'd remove it.